### PR TITLE
add send mail on new lead

### PR DESCRIPTION
**Pull request by CGary**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -20,7 +20,8 @@
       // enable the rule specifically for TypeScript files
       "files": ["*.ts", "*.tsx"],
       "rules": {
-        "@typescript-eslint/explicit-module-boundary-types": ["error"]
+        "@typescript-eslint/explicit-module-boundary-types": ["error"],
+        "prettier/prettier": 0
```
Comment (Carlos-YD) [inline .eslintrc:24]: Why are we overriding _prettier_ configurations?

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const corsOptions = {
+  origin: process.env.CORS_ORIGIN || '*'
+};
+app.use(cors(corsOptions));
+
+app.disable('x-powered-by');
+// Body parsing Middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
```
Comment (Carlos-YD) [inline src/server.ts:17]: Would be nice to just configure the app in this file and move the routes to another file(s), in the future for every new endpoint we will have to add more and more code here. Please check how to use **routers** in expressJS.

**Review by Carlos-YD — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (CGary) [reply .eslintrc]: I removed the line of code

File: src/server.ts (hunk 2 above)
Comment (CGary) [reply src/server.ts]: I added files with the routes of each endpoint

**Review by marcelo-yourday — APPROVED** [review]

**Review by kleyla — APPROVED** [review]

**Pull request merged**